tests/project/test_project_schemas.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from pydantic import ValidationError
+from snowflake.cli.api.project.schemas.project_definition import ProjectDefinition
+from snowflake.cli.api.project.schemas.snowpark.argument import Argument
+from snowflake.cli.api.project.schemas.snowpark.callable import FunctionSchema
+
+# TODO:
+# -test nice error message when update fails - this should be done by yaml loader, here we only check that error is raised
+# -test if runtime is passed correctly to Callable
+
```

Comment:
Intentional?

Reply:
No, some leftovers from development. Thanks for noticing!